src/js/components/Pagination/StyledPageControl.js

```diff
@@ -2,6 +2,43 @@ import styled from 'styled-components';
 
 import { Button } from '../Button';
 
+const BUTTON_STATES = ['default', 'active', 'disabled', 'hover', 'selected'];
+const defaultStyles = {
+  border: {
+    radius: '4px',
+  },
+  pad: '4px',
+};
+
+// Assemble button theme object for each button state
+const buildButtonStyle = (buttonTheme, kind) => {
+  const style = buttonTheme[kind];
+  BUTTON_STATES.forEach(state => {
+    if (state !== 'default') style[state] = buttonTheme[state][kind];
```

Comment:
is this all fine even if a buttonTheme[state] is undefined, or could errors be thrown?

Reply:
Returning undefined here is fine, because it is simply reconstructing what is defined in the theme (effectively creating a clone). If it is undefined in the theme, then it should also be undefined here.